components/about-us/sub-pages/Governance/start.tsx

```diff
@@ -1,4 +1,4 @@
-import { Stack, Flex, Box, Heading, Text, Image } from '@chakra-ui/react'
+import { Stack, Flex, Box, Heading, Text, Image, Link } from '@chakra-ui/react'
 import { FiArrowUpRight } from 'react-icons/fi'
 import TokenImg from '../../../../assets/banklesscoin.png'
 import Link from 'next/link'
```

Comment:
The problem is you're importing Chakra UI `Link` and Next.js `Link`. But this file doesn't use any routing, so I think you can just delete the Next.js `Link` import.

```suggestion

```